nlp_architect/procedures/transformers/que_ans.py

```diff
@@ -0,0 +1,197 @@
+# ******************************************************************************
```

Comment:
please rename file to something more semantic

Reply:
renaming to squad